Public/Src/Cache/ContentStore/Distributed/NuCache/RaidedRedisDatabase.cs

```diff
@@ -84,10 +87,12 @@ public async Task<BoolResult> ExecuteRedisAsync(OperationContext context, Func<R
         }
 
         /// <nodoc />
-        public async Task<(TResult primary, TResult secondary)> ExecuteRaidedAsync<TResult>(OperationContext context, Func<RedisDatabaseAdapter, Task<TResult>> executeAsync, bool concurrent = true, [CallerMemberName]string caller = null)
+        public async Task<(TResult primary, TResult secondary)> ExecuteRaidedAsync<TResult>(OperationContext context, Func<RedisDatabaseAdapter, CancellationToken, Task<TResult>> executeAsync, TimeSpan? retryWindow, bool concurrent = true, [CallerMemberName]string caller = null)

```

Comment:
>retryWindow [](start = 203, length = 11)

I think we should get rid of the retry window and just cancel retries once one of the operations finishes.

Reply:
The retry window is effectively a feature flag here.

Reply:
In standup today we discussed  to leave the retryWindow, and configure the amount later on to test its effects.